src/gui/ocsjob.h

```diff
@@ -110,6 +109,8 @@ class OcsJob : public AbstractNetworkJob
      */
     void addRawHeader(const QByteArray &headerName, const QByteArray &value);
 
+    QString getParamValue(const QString &key) const;
```

Comment:
@mgallien added, but unsure why we need it...

Reply:
because I want to make sure that if people call a `const` method that return a value, they use it to do something
not doing anything with the returned value is just a waste